src/filter.js

```diff
@@ -42,7 +42,12 @@ const EXCLUDED_MATERIAL_TYPES = [
 
 export default function (record, earliestCatalogTime) {
 	const leader = record.varFields.find(f => f.fieldTag === '_');
-	const materialType = record.materialType.code.trim();
+
+	const materialType = getMaterialType;
```

Comment:
Missing functions parentheses (The function is not called, only assigned as a value).

Reply:
Now it is    const materialType = getMaterialType();